app/helpers/application_helper.rb

```diff
@@ -14,4 +14,8 @@ def i18n_pluralize(word)
   def i18n_error_count(count)
     I18n.locale == :ja ? "#{count}件の#{t('views.common.error')}" : pluralize(count, t('views.common.error'))
   end
+
+  def show_user(instance)
+    instance.user.name == '' ? instance.user.email : instance.user.name
```

Comment:
```suggestion
    instance.user.name.present? ? instance.user.email : instance.user.name
```

present? を使うと「空のデータ」をもうちょっと幅広くカバーできます。
（ネットでこのメソッドの仕様を調べてみてください）

Reply:
present?が取り扱う「空のデータ」を学びました。
ご指摘いただいたとおりにpresent?で修正したところ、Rubocopに指摘されたので、以下に修正しました。
`user.name.presence || user.email`